### PR TITLE
Roslyn 2.9

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by filipw** [comment]
what does it bring?

**Comment by rchande** [comment]
@filipw Just investigating for breaking changes right now. We'll need to do this eventually.

File: src/OmniSharp.Http.Driver/app.config [hunk 1]
```diff
@@ -7,15 +7,15 @@
         <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
             <dependentAssembly>
                 <assemblyIdentity name="Microsoft.CodeAnalysis" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
-                <bindingRedirect oldVersion="0.0.0.0-2.7.0.0" newVersion="2.7.0.0"/>
+                <bindingRedirect oldVersion="0.0.0.0-2.9.0.0" newVersion="2.7.0.0"/>
```
Comment (DustinCampbell) [inline src/OmniSharp.Http.Driver/app.config]: Your binding redirects all need to set `newVersion` to `2.9.0.0`.

**Comment by DustinCampbell** [comment]
Your binding redirects are incomplete, which produces build warnings. After that, it looks like the following tests are failing on both Linux and OSX:

* OmniSharp.Roslyn.CSharp.Tests.HighlightFacts.HighlightStringInterpolation
* OmniSharp.Roslyn.CSharp.Tests.HighlightFacts.HighlightEntireFile
* OmniSharp.Roslyn.CSharp.Tests.HighlightFacts.HighlightSingleLine

**Pull request merged**